static/js/plotjs/stackSpectra.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable */
+const binsize = bin_slider.value;
+const fluxalph = binsize === 0 ? 1.0 : 0.1;
+
+for (let i = 0; i < n_labels; i++) {
+  const spectra = eval(`s${i}`);
```

Comment:
Is this eval necessary?

Reply:
I'll look into alternatives, but this how we've always accessed bokeh plot data sources within custom JS callbacks and I'm not aware of alternatives off the top of my head. Agreed it would be preferable not to use it

Reply:
Ok, I think I have a clean alternative, will update soon

Reply:
Ok @stefanv I believe the linting is much improved, not only here but in all the other Bokeh custom JS stuff. I chose to ignore the `no-undef` rule on a per-file basis because we often call variables dynamically passed in by Bokeh but not visible by ESLint when parsing the custom JS scripts. So that would just be a lot of lines with the "ignore this line because this variable will be there at runtime" behavior.

Reply:
Perfect! Thank you @kmshin1397.